change/@microsoft-teams-js-36a90bf1-c107-470b-b128-4291b1cdee38.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Updating authentication.notifySuccess/Failure logic to throw error in Frameless context",
```

Comment:
Is this correct? I don't see any changes to authentication.ts.

Based on the latest iteration, I would change this to:

"Added frameless unit tests for `authentication.notifySuccess` and `authentication.notifyFailure`"

(After all the CHANGELOG work I just did I'm being pretty picky about the wording/tense, etc. because I don't want to go back and clean them up 😉 )

Reply:
Sure, I'll update it